test/tests/bootstrap/test_redfish10_os_bootstrap_parallel.py

```diff
@@ -0,0 +1,196 @@
+'''
+Copyright 2017, Dell EMC
```

Comment:
OK, but the copyright statement I put here was from Rod. I think the statement from legal supersedes Rod. I will make the changes you suggest.